OracleDatabase/OracleRealApplicationClusters/README.md

```diff
@@ -0,0 +1,474 @@
+FOLLOWING FREELY AVAILABLE PATCH IS REQUIRED FOR THIS IMAGE TO WORK
+==================================
+- Download following patch from OTN and stage it on your machine.
+
+		- Patch# p27383741_122010_Linux-x86-64.zip
+	You can download this patch from following location:
+    
+    
+  	http://www.oracle.com/technetwork/database/database-technologies/clusterware/downloads/docker-4418413.html
+    
+  	Note: This patch required to run RAC on Docker in non-privileged mode.
+    
+Oracle RAC Database on Docker
+================================
+Sample Docker build files to facilitate installation, configuration, and environment setup for DevOps users. For more information about Oracle Database please see the Oracle Grid/RAC Database Online Documentation. (http://docs.oracle.com/en/database/)
+
+How to build and run
+======================
+This project offers sample Docker files for:
+
+- Oracle Database 12c Release 2 Grid Infrastructure (12.2.0.1.0) for Linux x86-64
+- Oracle Database 12c Release 2 (12.2.0.1.0) Enterprise Edition for Linux x86-64
+
+To assist in building the images, you can use the buildDockerImage.sh script. See below for instructions and usage.
+
+The buildDockerImage.sh script is just a utility shell script that performs MD5 checks and is an easy way for beginners to get started. Expert users are welcome to directly call docker build with their prefered set of parameters.
+
+IMPORTANT : You can build and run RAC containers on a single host or multiple hosts. To access the RAC DB on your network either use the Docker MACVLAN driver (see Docker documentation) or use Oracle Connection Manager. To Run RAC containers on Multi-Host, you must use the Docker MACVLAN driver and your network must be reachable on all the nodes for RAC containers. 
+
+For complete RAC setup, please execute the steps in the sections below per your environment:
+
+Section 1   : Pre-requisites for RAC on Docker 
+
+Section 2   : Building the Oracle RAC Database Docker Install Images
+
+Section 3   : Creating the Docker GI and RAC container
+
+Section 4   : Adding RAC Node using Docker container
+
+Section 5   : Connecting to RAC Database
+
+Section 6   : Env Variables for RAC 1st Node container
+
+Section 7   : Env Variables for RAC 2nd Node container
+
+Section 8   : License
+
+Section 9   : Copyright
+
+
+Section 1 : Pre-requsites for RAC on Docker
+================================
+IMPORTANT: You must make the changes specified in this section (customized for your environment) before you proceed to the next section.
+
+- For better performance, it is good to use BTRFS file system for Docker storage on the Docker host.
+- Install and configure Docker Engine on Oracle Linux 7 to run RAC on Docker.
```

Comment:
Please refer to "Oracle Container Runtime for Docker" and link to the documentation: https://docs.oracle.com/cd/E52668_01/E87205/html/index.html